source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -211,16 +211,16 @@ Location: https://YOUR_REDIRECT_URI?error=invalid_request
+|-------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `unauthorized_client`                     | In exceptional circumstances, such as a security incident, One Login may prevent users from logging in from your service. An `unauthorized_client` will be used in this case, and the error description will be `client deactivated`. You should consider showing the user a custom error page to explain that they cannot use your service at the moment and should try again later.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
To me deactivated suggests a permanent action.

Reply:
See this comment from Alex: https://gds.slack.com/archives/C05EW21TW3A/p1722605779657489?thread_ts=1722602980.421469&cid=C05EW21TW3A

I guess we could go for inactive?

Reply:
Could we include the word temporary?